src/relayclient/RelayProvider.ts

```diff
@@ -79,15 +79,19 @@ export class RelayProvider implements HttpProvider {
       console.log('calling sendAsync' + JSON.stringify(payload))
     }
     this.origProviderSend(payload, (error: Error | null, rpcResponse?: JsonRpcResponse): void => {
-      if (error != null) {
-        callback(error)
+      // Sometimes, ganache seems to return 'false' for 'no error' (breaking TypeScript declarations)
+      // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
+      if (error) {
```

Comment:
ok, but we might introduce `isTruthy` function - as a replacement for the above ugly eslint-disable ...